gensim/models/phrases.py

```diff
@@ -208,6 +208,10 @@ def load(cls, *args, **kwargs):
         """
         model = super(PhrasesTransformation, cls).load(*args, **kwargs)
         # update older models
+        # if value in phrasegrams dict is a tuple, load only the scores.
+        if len(model.__dict__['phrasegrams']):
+            if isinstance(list(model.__dict__['phrasegrams'].values())[0], tuple):
+                model.__dict__['phrasegrams'].update((k, v[1]) for k, v in model.__dict__['phrasegrams'].items())
```

Comment:
This is hard to read. Can you rephrase this with normal dict access syntax, and using better variable names? (not `k` and `v` and `v[1]`).

Also, changing the dict at the same time you're iterating over it sounds tricky. Probably safer to iterate over each value, and if it's a tuple, change it right away (in-place, without a full dict copy).

Reply:
Would there be any case where some values are tuple and some are not?
If not, we can check only one instance and based on that modify the entire dictionary.

Reply:
I don't think so -- all values should always be in the same format (all old, all new).

My suggestion was more around iteration and per-element assignment, so whether you check each value makes no difference (in fact, per-value checks may look cleaner, because there's no special case of "dictionary empty").